.github/workflows/nomad-requirements.yml

```diff
@@ -0,0 +1,56 @@
+name: Validate Dependencies Compatibility
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    # Run workflow only when there are changes in pyproject.toml or dev-requirements.txt
+    # paths:
+    # - 'pyproject.toml'
```

Comment:
From my point of view it would make sense to activate these. Or are there still some problems when using these lines?

Reply:
I intentionally kept this open because it wouldn't run the CI/CD in this branch. Now that everything is fixed, I have reactivated them.